Libplanet/PoS/Control/ValidatorCtrl.cs

```diff
@@ -0,0 +1,205 @@
+using System;
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Assets;
+
+namespace Libplanet.PoS
+{
+    internal static class ValidatorCtrl
+    {
+        internal static Validator GetValidatorByAddr(
+            IAccountStateDelta states,
+            Address validatorAddress)
+        {
+            IValue? serializedValidator = states.GetState(validatorAddress);
+            if (serializedValidator == null)
+            {
+                throw new InvalidOperationException();
+            }
+
+            Validator validator = new Validator(serializedValidator);
+            return validator;
+        }
```

Comment:
How about something like this to keep the behavior similar to `IAccountStateDelta.GetState()`?
```suggestion
        internal static Validator? GetValidatorByAddr(
            IAccountStateDelta states,
            Address validatorAddress)
        {
            if (states.GetState(validatorAddress) is { } value)
            {
                return new Validator(value);
            }
            return null;
        }
```

Reply:
Seems better